src/h5web/providers/utils.ts

```diff
@@ -55,6 +56,10 @@ export function isBaseType(type: HDF5Type): type is HDF5BaseType {
   );
 }
 
+export function isNumericType(type: HDF5Type): type is HDF5NumericType {
+  return isBaseType(type) && type.class !== HDF5TypeClass.String;
```

Comment:
I would change the typeguard into `return type.class === HDF5TypeClass.Integer || type.class === HDF5TypeClass.Float` for consistency with `HDF5NumericType` and in case `BaseType` needs to be extended.

Reply:
Ah yeah I think I did that because `isBaseType` also makes sure that `type` is not a `string`. Perhaps I can change `isBaseType` to use `isNumericType`, instead? It would make more sense this way.

Reply:
Neh, doesn't work, I'll just duplicate the logic.

Reply:
I am not sure. My opinion is that it *happens* that `NumericType`is `BaseType` without `String` but it depends on what is contained in `BaseType`.

I feel that putting that `Numeric` is either `Integer` or `Float` is clearer.